src/App.css

```diff
@@ -1,3 +1,80 @@
+@import url('https://fonts.googleapis.com/css2?family=Harmattan:wght@400;700&display=swap');
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  font-family: 'Harmattan', sans-serif;
+  font-weight: normal;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family:
+    'Harmattan',
+    'Courier New',
+    monospace;
+}
+
 .App {
+  display: flex;
+  align-items: center;
+  justify-content: center;
   text-align: center;
+  height: 100vh;
+}
+
+#calculator {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  grid-template-columns: repeat(4, 100px);
+  grid-template-rows: minmax(100px, auto) repeat(5, 60px);
+}
+
+#display {
+  grid-column: 1 / -1;
+  background-color: rgba(0, 0, 0, 0.75);
```

Comment:
- [ ] You need to stick with the original design including background colors. Kindly compare the background color of the display in your design and the original one. :pray: 

![image](https://user-images.githubusercontent.com/64894927/138115012-346997cc-3253-481e-92ad-7bb1edd1257e.png)
![image](https://user-images.githubusercontent.com/64894927/138115050-d0059772-3383-4119-8e1c-6a6f0963d74a.png)